lib/main.coffee

```diff
@@ -2,6 +2,16 @@
 helpers = require 'atom-linter'
 path = require 'path'
 
+cmpVersions = (v1, v2) ->
```

Comment:
can we have some doc comment or so? Also maybe it should be rather a function that returns a boolean, i.e. `isGreaterVersion` or so?